requirements/common.txt

```diff
@@ -21,7 +21,7 @@ django-sass-processor==0.5.1
 django-sekizai==0.10.0
 Django-Select2==5.8.9
 djangoshop-stripe==0.2.0
-django-treebeard==4.0
+https://github.com/fladi/django-treebeard/archive/django-1.10.zip
```

Comment:
Please don't pin to a branch. If we cannot pin to a PyPI release, we should pin to an exact commit.